README.md

```diff
@@ -116,29 +108,23 @@ Then go to `http://localhost:8080` in your web browser.
 
 Having problems?  Check out our [Troubleshooting guide](TROUBLESHOOTING.md).
 
-## CUTR Release Process
+## MobilityData Release Process
 
 **Snapshots**
 
-We've set up a Maven repository to hold the snapshot artifacts from this project in a Github project - [cutr-mvn-repo](https://github.com/CUTR-at-USF/cutr-mvn-repo).
-
-At CUTR, we should run the following at the command-line to create a new artifact:
-~~~
-mvn -Dgpg.skip -DaltDeploymentRepository=cutr-snapshots::default::file:"/Git Projects/cutr-mvn-repo/snapshots" clean deploy
-~~~
-
-Then commit using Git and push new artifacts to Github.
+We've set up a Maven repository on GitHub Packages to hold the snapshot artifacts from this project - [GTFS Realtime Validator Packages](https://github.com/orgs/MobilityData/packages?repo_name=gtfs-realtime-validator).
 
 If you want to include snapshot releases in your project, you'll need to add the following to the `pom.xml` of the project you want to use it in:
 
 ~~~
-<!-- CUTR SNAPSHOTs/RELEASES -->
-<repositories>
-    <repository>
-        <id>cutr-snapshots</id>
-        <url>https://raw.githubusercontent.com/CUTR-at-USF/cutr-mvn-repo/master/snapshots</url>
-    </repository>        
-</repositories>
+<!-- MobilityData SNAPSHOTs/RELEASES -->
```

Comment:
This should be a `<repositories>` block  like in the original file, not `<dependencies>`. The `<repositories>` block is needed because the artifacts aren't hosted on Maven Central (where all maven projects look by default) but instead on GitHub. GitHub also makes this more complicated because you need to authenticate with GitHub.

Here's another project that I've written instructions for for this:
https://github.com/barbeau/earth-gravitational-model

You should be able to copy/paste and then change the repositories section, and make sure you include the text with the link to authenticate with GitHub too.

I think for now we can do the above to just update the instructions for using the snapshots and then cross the bridge for releases when we get there, which ideally should be on Maven Central.

Reply:
Right. I updated it in both `README.md` and `gtfs-realtime-validator-lib/README.md`.